crates/sqllib/README.md

```diff
@@ -1,2 +0,0 @@
-This crate contains Rust types and functions used by the Feldera SQL
```

Comment:
why remove this?

Reply:
it gave a warning when publishing the crate because it has a readme but it also inherits the readme from the root